src/vm/eventpipe.cpp

```diff
@@ -38,6 +40,145 @@ extern "C" void InitProvidersAndEvents();
 extern "C" void InitProvidersAndEvents();
 #endif
 
+EventPipeEventPayload::EventPipeEventPayload(BYTE *pData, unsigned int length)
+{
+    CONTRACTL
+    {
+        NOTHROW;
+        GC_NOTRIGGER;
+        MODE_ANY;
+    }
+    CONTRACTL_END;
+
+    m_pData = pData;
+    m_pBlobs = NULL;
+    m_blobCount = 0;
+    m_performedAllocation = false;
```

Comment:
Nit: Rename m_performedAllocation to m_allocatedData to make it clear what was allocated.